api/application/client_test.go

```diff
@@ -30,7 +30,7 @@ type applicationSuite struct {
 var _ = gc.Suite(&applicationSuite{})
 
 func newClient(f basetesting.APICallerFunc) *application.Client {
-	return application.NewClient(basetesting.BestVersionCaller{f, 6})
+	return application.NewClient(basetesting.BestVersionCaller{f, 8})
```

Comment:
Wow... we've jumped from 6 to 8? what happenned to testing 7? It ate 9 and is now in sin-bin?

Reply:
lol. We were up to 7 but the test didn't reflect it